contents/handbook/people/team-structure/core-experience.md

```diff
@@ -11,27 +11,40 @@ hideAnchor: true
 - [Paolo D'Amico](/handbook/people/team#paolo-damico-product-team) (Product Manager)
 - [Sam Winslow](/handbook/people/team#sam-winslow-full-stack-engineer) (Full Stack Engineer)
 - [Li Yi Yu]((/handbook/people/team/#li-yi-yu-software-engineer)) (Full Stack Engineer)
+- Alex Kim (Full Stack Engineer)
 
 ## Mission
 
-To create the smoothest user experience.
+To create the smoothest experience when discovering insights about products and its end users.
 
 ## Responsibilities
+List below is prioritized.
 
-- Ensuring performant and clear user experience across all analytics functionality
-- Build and improve frontend features across the app
-- Extending feature set as suggested by requests, by our own ideas, and by need for parity with other platforms
+1. Reaching parity in terms of features, performance and user experience with leading product analytics solutions, including:
+   1. Extending core feature set (particularly on the frontend) as informed by dogfooding, user feedback, experimentation and strategic decisions.
+   2. Extending user experience to enable product analytics for non-technical users. 
+2. Maintaining (and improving where applicable) supporting activities for the core experience (this includes: cohorts, persons, sessions *[not session recording]*, annotations, settings, authentication, account management). 
 
 
 ## Customer
-
-- Product managers at scale-ups
+In general, any PostHog user, whether they're an engineer, product manager or business person, should be able to effectively analyze their product. However, our **focus in the short-term is product managers at scale-ups that fit our focus customer persona.**
```

Comment:
@paolodamico My thinking behind having a single well defined customer is to really focus our efforts on this. I'm not sure broading it out like this will help?

Reply:
Agree on having a specific focus to make sure we're successful with them. Clarified the wording a bit to make it clear we still want to support technical users well.